Generator/OdmGenerator.php

```diff
@@ -72,7 +72,14 @@ protected function generateMappingDocumentFiles(OutputInterface $output, BundleM
                 $output->writeln(sprintf('   ~ <info>%s</info>', $fileName));
             } else {
                 $output->writeln(sprintf('   + <info>%s</info>', $fileName));
-                copy($src_file, $dest_file);
+
+                $mappingEntityTemplate = file_get_contents($src_file);
```

Comment:
The file name was not introduced by me. We use the underscore notation for several vars in this bundle. Maybe we should fix this with another PR.